src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,157 @@
+  }
+
+  .logo + p {
+    position: relative;
+    margin-bottom: 0;
+    margin-top: 1.5rem;
+    padding-top: 1.5rem;
+    text-align: center;
+
+    &::before {
+      position: absolute;
+      content: "";
+      width: 5rem;
+      height: 0;
+      border-bottom: 1px solid transparentize($white,0.5);
+      top: 0%;
+      left: calc( (100% - 5rem) / 2 );
+    }
+
+    @include media-breakpoint-up(md) {
+      padding-top: 0;
+      margin-top: 0;
+      padding-left: 1.5rem;
+      margin-left: 1.5rem;
+      text-align: left;
+
+      &::before {
+        width: 0;
+        height: 70%;
+        border-top: 0;
+        border-left: 1px solid transparentize($white,0.5);
+        top: 15%;
+        left: 0;
+      }   
```

Comment:
I'm seeing trailing whitespace here. Can you set your editor to delete that on save?